examples/utils.go

```diff
@@ -0,0 +1,33 @@
+package examples
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/bitcoin-sv/spv-wallet-go-client/xpriv"
+)
+
+// 'xPriv' | 'xPub' | 'adminKey' | 'Paymail'
+type keyType string
+
+func ErrMessage(key keyType) string {
+	return fmt.Sprintf("Please provide a valid %s.", key)
+}
+
+func GenerateKeys() xpriv.KeyWithMnemonic {
+	keys, err := xpriv.Generate()
+	if err != nil {
+		fmt.Println(err)
+		os.Exit(1)
+	}
+
+	return keys
+}
+
```

Comment:
I would prefer to put this code of generation directly to the `generate_keys.go` example. 
Thanks to that, the new user would not have to search "what does the `examples.GenerateKeys()` do".

Reply:
I moved it to `example_keys.go`, the problem with moving it to `generate_keys.go` is that this file is in package `main` and in it's own sub-directory and the function cannot be imported elsewhere because of this. 
`example_keys.go` does not have this problem (package `examples` and `root` directory)